tests/zfs-tests/include/libtest.shlib

```diff
@@ -37,7 +37,7 @@
 
 function is_linux
 {
-	if [[ $($UNAME -o) == "GNU/Linux" ]]; then
+    if [[ $($UNAME -o) == "GNU/Linux" ]]; then
```

Comment:
The -o option to uname is not standardized across OSes, so you'll need to check that it fails outright.
How about this?
```
A=$($UNAME -o 2>/dev/null) || return 1
[[ $A == "GNU/Linux" ]] || return 1
return 0
```